src/shared/routes.ts

```diff
@@ -31,7 +31,7 @@ export const routes: IRouteProps[] = [
     component: Releases,
   },
   {
-    path: `/news_item/:index`,
+    path: `/news_item/:title`,
```

Comment:
Looks a bit ugly with the underscore, i would simply call it `/news/:title`. Dont think it would cause any routing problems.

Reply:
I can do that.